### PR TITLE
Add new promo card Bactoviral Research

### LINKED ISSUE #3268: Add card Bactoviral Research
Details at https://boardgamegeek.com/thread/2655140/summer-2021-season-promo

Can't tell if this is the missing X35 or not. Would assume X37 for now.

(Note: the app should try to make sure not to add any fan card that collects microbes but lacks a microbe tag, as it would then be unclear what this card is supposed to do (same with animals and Eos Chasma). Tangent: also a fan card that collects any *other* kind of resource should *not* have a microbe/animal tag because then it's unclear what Viral Enhancers should do.)

### PR DESCRIPTION
**Pull request by vsrisuknimit**
This can close #3268 .

image

### CODE REVIEW

**Review by nwai90 — APPROVED** [review]

**Review by bafolts — APPROVED** [review]

**Comment by vsrisuknimit** [comment]
@bafolts We probably should wait for @kevinb9n to confirm the accuracy. I cannot see the video. Not a patron. So I am not sure if this information is accurate.

**Comment by bafolts** [comment]
> @bafolts We probably should wait for @kevinb9n to confirm the accuracy. I cannot see the video. Not a patron. So I am not sure if this information is accurate.

Not sure. If a PR is not ready to be merged can you use `do not merge` label?

**Comment by kevinb9n** [comment]
Hmm, it's an unlisted video, but I think you can still see it?  https://www.youtube.com/watch?v=Bxh4rBw5xKo

You'll laugh when you see how little I had to go on to guess the text, but the icons are clear.

File: src/cards/promo/BactoviralResearch.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+export class BactoviralResearch extends Card implements IProjectCard {
+  constructor() {
+    super({
+      cardType: CardType.AUTOMATED,
+      name: CardName.BACTOVIRAL_RESEARCH,
+      tags: [Tags.MICROBE, Tags.SCIENCE],
+      cost: 10,
+      metadata: {
+        cardNumber: 'X37',
```
Comment (kevinb9n) [inline src/cards/promo/BactoviralResearch.ts:20]: maybe add `// unconfirmed`

Comment (kevinb9n) [reply src/cards/promo/BactoviralResearch.ts:20]: Oh, I thought the PR was unmerged - please ignore this.

**Comment by vsrisuknimit** [comment]
> Hmm, it's an unlisted video, but I think you can still see it? https://www.youtube.com/watch?v=Bxh4rBw5xKo
> (I'll probably be nice and remove that a bit later)
> 
> You'll laugh when you see how little I had to go on to guess the text, but the icons are clear.

That seems accurate.

**Comment by vsrisuknimit** [comment]
> > @bafolts We probably should wait for @kevinb9n to confirm the accuracy. I cannot see the video. Not a patron. So I am not sure if this information is accurate.
> 
> Not sure. If a PR is not ready to be merged can you use `do not merge` label?

@bafolts Sorry. Just thought @kevinb9n would have a chance to look before a merge. I tagged him. I will put on the label next time.

**Pull request merged**